scripts/repo_metrics/track_metrics.py

```diff
@@ -1,6 +1,13 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+import sys
+# Need to append a full path instead of relative path.
+# This seems to be an issue from Azure DevOps command line task.
+# NOTE this does not affect running directly in the shell.
+sys.path.append('cicdworkdir')
```

Comment:
I'm normally against hardcoding, but I think it is important to have the tracking system working every day. So I will approve and then add some suggestions, if you want to add them now or later is up to you. 
Suggestions:
1) not sure how we are executing this, but I'm sure we can add to sys path the [current folder](https://github.com/miguelgfierro/codebase/blob/master/python/system/paths.py#L15) or maybe the [parent folder](https://github.com/miguelgfierro/codebase/blob/master/python/system/paths.py#L27).
2) It would be good to add a parameter with the date, by default it could be `now()` but let's say we want to add an event that didn't happen today, with the current code we can't do it.

An additional comment, we should record events to make sure we are able to match metrics with marketing actions that we do with respect of time. One example of event that we should record is the day we opensource the repo, when we have a new release or when we publish a new blog

Reply:
> not sure how we are executing this, but I'm sure we can add to sys path the current folder or maybe the parent folder.
We execute the python script directly with the command line task in the Azure DevOps pipeline, but the problem is that the command line script somehow cannot recognize relative paths such as `.` or `../..`. Thus I add it manually. But I think it would be definitely better to avoid hardcoding - now the guy is just to make sure things work :)
> An additional comment, we should record events to make sure we are able to match metrics with marketing actions that we do with respect of time.
For this one, the script should be manually executed, right? Yea we can do that definitely.